newIDE/app/src/MainFrame/EditorContainers/StartPage/Carousel.css

```diff
@@ -0,0 +1,19 @@
+ul.MuiGridList-root {
```

Comment:
Interesting, is there a risk we conflict with other usage of "MuiGridList" in the future?

Reply:
Well it would if this style is present in all components, I am not sure about that though: if there is a `MuiGridList` in a component, and this carousel in a dialog for example, will it affect the first?
Anyway, I can add an id/data to the div that wraps the `MuiGridList` and change the CSS to have a `>`

Reply:
I think so, the inclusion of the Carousel.css will *not* be scoped to the component (that's also why we avoid using raw CSS files in the app, they are global and will suffer from traditional CSS flaws: unability to find what is unused/used, risk of impacting other components if you're not careful).

Take a look at if:
1. inline styles would work (passing `style={styles.xxx}` to the component). It's usually good enough for most stuff.
2. if you need to use advanced CSS or modify embedded tags, then you can use Material-UI v4 styling solution. See an example here: https://github.com/4ian/GDevelop/blob/2a386cdcf168437a81b4476eb96d2ec79cb34270/newIDE/app/src/UI/SemiControlledAutoComplete.js#L80-L98
  - This gives you a useStyles hook than you can call and pass the resulting classes to the component. See https://v4.mui.com/api/grid-list/#gridlist-api for this specific component
3. As a last resort, a few global CSS classes can work! :) In this case it's surely safer to use a classname and change the CSS to have a `>`
  
Note that the makeStyles API of Material-UI will change in MUI v5, so between this and a CSS file with a proper classname, I'm not sure what I prefer. I usually just try hard at using only inline styles (to keep the styling inside the component with 0 risks of it leaking outside - the style IS the component) and use the other two solutions as last resorts.

Reply:
I didn't know you could specify pseudo elements and states styles with makeStyles! That's great with it!